### ISSUE TITLE
Doesn't work in 4.3

### ISSUE THREAD

**Issue by darkhog** [blackears/cyclopsLevelBuilder#191]
When loading a project containing Cyclops in 4.3:

```
  res://addons/cyclops_level_builder/exporter/exporter_gltf_wizard.gd:26 - Parse Error: Class "ExporterGltfWizard" hides a global script class.
  modules/gdscript/gdscript.cpp:2936 - Failed to load script "res://addons/cyclops_level_builder/exporter/exporter_gltf_wizard.gd" with error "Parse error". (User)
  res://addons/cyclops_level_builder/exporter/exporter_godot_scene_wizard.gd:26 - Parse Error: Class "ExporterGodotSceneWizard" hides a global script class.
  modules/gdscript/gdscript.cpp:2936 - Failed to load script "res://addons/cyclops_level_builder/exporter/exporter_godot_scene_wizard.gd" with error "Parse error". (User)
```

When trying to play the game:

```
Could not resolve external class member "data".
```

in convexblockdata.gd:99

For some weird reason my scenes look fine in editor (blocks didn't disappear or anything).

**Comment by blackears** [comment]
Similar to https://github.com/blackears/cyclopsLevelBuilder/issues/192

**Comment by darkhog** [comment]
So just to be sure, check out master branch, run create\_addon\_archive.py, import the result thing and everything will be fine?

**Comment by blackears** [comment]
Yes.  Just replace everyting in the /addon folder.  You may need to reload twice.

**Comment by darkhog** [comment]
Um... finally got around to doing that and the exact same thing still happens. This issue is not solved.

**Comment by blackears** [comment]
I'm guessing that your build is out of date.  The first two errors you mention are due to Godot not registering the class names correctly.  And DataVectorByte definitely has a field named `data`.  I've had to deal with similar situations a few times where Godot just refuses to update how it registers (or "compiles" or whatever it is doing) different classes, even after restarting.  Sometimes deleting the .godot folder and restarting will help.  But I've had to completely rebuild my project once too, so it's possible you might need to rebuild your project in a fresh directory.

I can run my test program without error so I'm thinking this is perhaps specific to your machine.  Godot doesn't compile the scripts - it's doing something else and caching it which causes weird inconsistencies.  I was hoping this would be fixed in 4.3, but apparently not.
